UnicornEngine/video/include/unicorn/video/vulkan/Image.hpp

```diff
@@ -0,0 +1,101 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_VULKAN_IMAGE_HPP
+#define UNICORN_VIDEO_VULKAN_IMAGE_HPP
+
+#include <vulkan/vulkan.hpp>
+#include <unicorn/video/vulkan/Buffer.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+namespace vulkan
+{
+/**
+ * @brief Wrapper for Vulkan image
+ */
+class Image
+{
+public:
+    /**
+   * @brief Creates image
+   * @param device device for allocation
+   * @param physicalDevice for memory type search and hardware allocation
```

Comment:
change the order of `device` and `physicalDevice` in doxy comment according to argument order in method itself